doc/LocalVMSetup.md

```diff
@@ -16,9 +17,24 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
+## 4. You may need to configure your network settings
+run a test to check if you are connected to the network 
```

Comment:
Start the sentence with a capital letter. Consider adding a colon `:` at the end here.